types/jsreport/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for jsreport 2.9
+// Project: https://github.com/jsreport/jsreport
+// Definitions by: pofider <https://github.com/pofider>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// import all availible types for jsreport included extensions
+import JsReportChromePdf = require('jsreport-chrome-pdf');
+import JsReportJsRender = require('jsreport-jsrender');
+import JReportHtmlToXlsx = require('jsreport-html-to-xlsx');
+import JsReportXlsx = require('jsreport-xlsx');
+
+// just reexport types from core
+import JsReport = require('jsreport-core');
+export = JsReport;
```

Comment:
@pofider 
I'm not sure if that is a proper way to reflect the module contrat, can you check this?
https://github.com/jsreport/jsreport/blob/master/index.js#L62-L75
(IMO module exports its own function and re-exports other types)

Reply:
Thank you for reviewing. 
We were planning to remove the other exports in the next major, so I was thinking there is no point in adding types now.
I just wanted to re-export the types that are exported in [jsreport-core](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jsreport-core/index.d.ts) package. 
If it's an internal rule of the repository to cover all exports, I can add it temporarily for now.

Reply:
> If it's an internal rule of the repository to cover all exports, I can add it temporarily for now

Nope, only for a concern of providing same experience for users and reflecting the native module properly (in short, stick with the goal of the DT repo: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#L3).
I see the point. Also you own both implementations, so it's up to you (and for DT maintainer for final touch).